tests/golden_inputs/test_intf.yis

```diff
@@ -0,0 +1,45 @@
+# -*- yaml -*-
```

Comment:
Shouldn't there be changes to the associated test_intf.html?

Reply:
added